app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -212,6 +212,13 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def queue_callback(state, msg, result)
+    _log.info("queue_callback called with pod name #{state}, #{msg}, #{result}")
+    if state == "timeout"
+      queue_signal(:abort_job, "Job Timeout: #{msg}", "error")
```

Comment:
```Job Timeout: #{msg}``` will show up in the UI ("All vm and container analysis tasks") and we had issues where long messages broke the screen in the past. Please make sure that is alright